test/integration/proxy_test.js

```diff
@@ -715,7 +715,7 @@ test('reroutes retry to remote', function t(assert) {
             retrySchedule: [1]
         }, function onRequest(err, res) {
             assert.ifError(err, 'no error occurs');
-            assert.equal(res.body, 'rerouted', 'response from rerouted request is correct');
+            assert.equal(res.body.toString(), 'rerouted', 'response from rerouted request is correct');
```

Comment:
I agree. If the API did not change, we shouldn't change the test. Can you help me understand why this test changed?